source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -115,48 +123,95 @@ Table 3.p (below): List of Data_Type Values
      - Reports
 
    * - Article
-     - An article, typically published in a journal or reference work.
+     - An article from a journal, or an article available as a standalone content item not aggregated into a title, for example in an institutional repository.
      - Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, IR\ |br|\ |lb|
-       PR_P1, IR_A1
+     - PR, IR
+
+   * - Audiovisual
+     - A form of multimedia, typically describing video content.
+     - Multimedia\ |br|\ |lb|
+       Multimedia_Collection
```

Comment:
Since Full_Content_Database might include multimedia content, this Data_Type and the other multimedia Data_Types also should list Full_Content_Database?

Reply:
correct, added